cirq/work/observable_measurement.py

```diff
@@ -211,3 +269,148 @@ def _subdivide_meas_specs(
         repetitions //= 2
 
     return flippy_mspecs, repetitions
+
+
+def _to_sweep(param_tuples):
+    """Turn param tuples into a sweep."""
+    to_sweep = [dict(pt) for pt in param_tuples]
+    to_sweep = study.to_sweep(to_sweep)
+    return to_sweep
+
+
+def _get_qubits(max_settings: Iterable[InitObsSetting]):
+    """Helper function to find all the qubits in a suite of settings used
+    in `measure_grouped_settings`.
+
+    Qubits are returned in sorted order.
+    """
+    qubits = set()
+    for max_setting in max_settings:
+        max_in_st = max_setting.init_state
+        qubits |= set(max_in_st.qubits)
```

Comment:
Could be done with list comprehension: 
```
return sorted({q for ms in max_settings for q in ms.qubits})
```

Reply:
I always shy away from these double comprehensions because I can never remember how they work! Must think "odometer". Changed, and removed this helper function since it's now one line (line 337).